lib/EarlyFunctionAnalysis/EarlyFunctionAnalysis.cpp

```diff
@@ -613,6 +616,44 @@ void FunctionAnalysisResults::importModel() {
   }
 }
 
+void FunctionAnalysisResults::importDynamicSymbols() {
+  llvm::Function &Root = *M.getFunction("root");
+
+  for (BasicBlock &BB : Root) {
+    if (getType(&BB) != BlockType::JumpTargetBlock)
+      continue;
+
+    auto *SymbolNameString = hasDynamicSymbol(&BB);
+    if (not SymbolNameString)
+      continue;
+
+    auto [Entry, _] = getPC(&*BB.begin());
+
+    // Already imported?
+    if (FunctionsBucket.find(Entry) != FunctionsBucket.end())
+      continue;
```

Comment:
Mmmh, so we're giving precedence to locally defined functions uh?
Might make sense, but please put a comment that clearly states this.

Reply:
Now that you make me think about it, I'm not sure about this decision and why I exactly I decided to do so. Might make more sense removing this check. Works?

Reply:
Who takes precedence? Local functions or dynamic functions?
That's the decision to make.

Reply:
Makes sense, then please make sure the code reflects this and add a comment about this so we know it was intentional and not something we overlooked.

Reply:
Sounds good!